cloud/vsphere/provisioner/govmomi/update.go

```diff
@@ -65,6 +68,7 @@ func (vc *Provisioner) updateIP(cluster *clusterv1.Cluster, machine *clusterv1.M
 	if nmachine.ObjectMeta.Annotations == nil {
 		nmachine.ObjectMeta.Annotations = make(map[string]string)
 	}
+	glog.Infof("updateIP - IP = %s", vmIP)
```

Comment:
switch to using `glog.V(4).Infof(...` so that we can selectively emit more or less logs